src/components/Filter.js

```diff
@@ -138,7 +143,6 @@ function Filter() {
       <div>
```

Comment:
consider adding className to div, so its easier to inspect in browser when we're testing out things :)